v2/pkg/protocols/http/http_method_types.go

```diff
@@ -0,0 +1,112 @@
+package http
+
+import (
+	"encoding/json"
+	"errors"
+	"strings"
+
+	"github.com/alecthomas/jsonschema"
+)
+
+// HttpMethodType is the type of the method specified
+type HttpMethodType int
+
+const (
+	HttpGet HttpMethodType = iota + 1
```

Comment:
Http should be `HTTP`, all caps, which is the convention when writing it.

Reply:
Indeed, https://github.com/golang/go/wiki/CodeReviewComments#initialisms recommends this, but personally I don't agree with it. 

>Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". As an example: ServeHTTP not ServeHttp. For identifiers with multiple initialized "words", use for example "xmlHTTPRequest" or "XMLHTTPRequest".

`HttpGet` is more readable than `HTTPGet`, not to mention if the variable name would need to contain multiple glued in acronyms (e.g. `XMLHTTPRequest` vs `XmlHttpRequest` or `NATOHTTPSURL` vs `NatoHttpsUrl`)

Reply:
It's funny how Google has one recommendation/naming standard for Go (acronyms have to be all caps), but in Java they prefer otherwise https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

Reply:
I think we should move forward with `HTTP` in all caps as that is the convention. We're following the Go Style Guide so it makes sense to keep it caps.

Reply:
Following the Go Style Guide and  keeping HTTP in all caps .